packages/docz-tools/src/gatsby-theme-docz/components/Sidebar/styles.ts

```diff
@@ -30,7 +30,7 @@ export const input = {
   appearance: "none",
   border: 0,
   width: "100%",
-  bg: "blue",
+  bg: "greyBlueLightest",
```

Comment:
These next 3 changes are re-styling search as it used to use `blue`

Reply:
![image](https://user-images.githubusercontent.com/39704901/126719755-6b889a74-3ff2-45ad-9d2d-82897076af89.png)